code/game/objects/items/weapons/cosmetics.dm

```diff
@@ -95,6 +95,10 @@
 			if(!get_location_accessible(H, "mouth"))
 				user << "<span class='warning'>The mask is in the way.</span>"
 				return
+			if(H.species && H.species.flags & ALL_RPARTS) //If the target is of a species that can have prosthetic heads, but doesn't have one...

```

Comment:
Thanks, didn't think about that. I just figured it'd be good form to have it in there haha